suggest-cats.js

```diff
@@ -82,10 +88,17 @@ const categorize = (transaction) => {
 
     for (i = 0; i < res.data.length; i++) {
       console.log(`transaction: ${res.data[i].imported_payee}`);
-      messages.push(categorize(res.data[i].imported_payee))
-      const response = await chat(messages);
-      console.log(`response: ${response.choices[0].message.content}`);
-      messages.pop();
+      if (res.data[i].imported_payee) {
+        messages.push(categorize(res.data[i].imported_payee))
+        const response = await chat(messages);
+        console.log(`response: ${response.choices[0].message.content}`);
+        
+        let newCategory = existing_categories.find(cat => cat.name === response.choices[0].message.content);
+        if (newCategory) {
+          await api.updateTransaction(res.data[i].id, { category: newCategory.id });
+        }
+        messages.pop();
+      }
```

Comment:
Can you remove this from this PR?  I planned to allow updates, but I wanted to make those optional and have the possibility of being interactive (to confirm each one before updating).